packages/frontend/src/components/wallet/ExploreApps.js

```diff
@@ -111,34 +114,34 @@ const ExploreApps = () => {
         <Container>
             <div className='apps-wrapper'>
                 <div>
-                    <img src={HashRushLogo} alt='Hash Rush'/>
-                    <img src={FluxLogo} alt='Flux'/>
-                    <img src={ParasLogo} alt='Paras'/>
-                    <img src={RefLogo} alt='Ref Finance'/>
-                    <img src={MintbaseLogo} alt='Mintbase'/>
-                    <img src={BerryClubLogo} alt='Berryclub'/>
+                    <img src={HashRushLogo} alt='Hash Rush' />
+                    <img src={FluxLogo} alt='Flux' />
+                    <img src={ParasLogo} alt='Paras' />
+                    <img src={RefLogo} alt='Ref Finance' />
+                    <img src={MintbaseLogo} alt='Mintbase' />
+                    <img src={BerryClubLogo} alt='Berryclub' />
                 </div>
             </div>
             <h2>
-                <Translate id='exploreApps.title'/>
+                <Translate id='exploreApps.title' />
             </h2>
             <div className='desc'>
-                <Translate id='exploreApps.desc'/>
+                <Translate id='exploreApps.desc' />
             </div>
             <div className='buttons'>
                 <FormButton
-                    linkTo={EXPLORE_APPS_URL}
+                    linkTo={SHOW_EXPLORE_PAGE ? EXPLORE_APPS_URL : AWESOME_NEAR_URL}
```

Comment:
Can we please skip any changes to the awesomenear.com url since it's already pointing to the main page 🙏